src/formatter/helpers/issue_helpers.js

```diff
@@ -93,6 +93,18 @@ function formatStep({
   }
   text += '\n'
 
+  if (Array.isArray(testStep.attachments)) {
```

Comment:
This can be simplified to `if (testStep.attachments) {` as it should always be an array or null.